### PR TITLE
Removes facial disfigurement

### PR DESCRIPTION
**Pull request by Doubleumc**
# About the pull request

Removes facial disfigurement.

# Explain why it's good for the game

Facial disfigurement is a roleplay tool, one which doesn't really have a place in CM due to its rules around messing with other people's IDs. I have never seen any sort of impersonation plot or deeper engagement from facial disfigurement. The most I've seen is "OH GOD MY FACE!" and then the marine gets it fixed, or ignores it.

So why not leave it as-is? It causes no harm. While leaving it is harmless it is just one more aspect of "standard" SS13 that doesn't fit with CM. One more piece of code that has to be maintained through changes and refactors. One more thing to consider when trying to understand what the code does. In the same vein as the removal of walk/run mode and aim mode.
# Testing Photographs and Procedure
Boots without issue.  Facial damage, yautja skinning, and var editing still work minus the facial disfigurement.

# Changelog
:cl:
del: Removed facial disfigurement
/:cl:

### CODE REVIEW

**Comment by ihatethisengine** [comment]
Can't you just remove "making you unknown" part without removing this feature as just a fluff and maybe a thing you can read in examine text? I understand that Unknown (as John Doe) is annoying, but this is the only part that is annoying, we can keep the rest for _at least some_ roleplay value.

**Comment by GrrrKitten** [comment]
> Can't you just remove "making you unknown" part without removing this feature as just a fluff and maybe a thing you can read in examine text? I understand that Unknown (as John Doe) is annoying, but this is the only part that is annoying, we can keep the rest for _at least some_ roleplay value.

It really adds no Roleplay value that you couldn’t get by added a category of “they have significant burn scars on their [list(burntlimb’s)]”.

That would need to be a seperate PR tho, but could be a significantly simpler fluff system

**Comment by Doubleumc** [comment]
That is entirely possible and could be easily done. However, I still believe removal is preferable for the reasons I outlined above. People just don't engage with it, and a feature people don't engage with is just unnecessary code.

**Comment by PhantomEpicness** [comment]
ancient SS13 leftovers have no place here. Maybe if a medical system rework comes around (kenshimed...), then maybe we can reconsider putting this back

**Comment by zzzmike** [comment]
> Can't you just remove "making you unknown" part without removing this feature as just a fluff and maybe a thing you can read in examine text? I understand that Unknown (as John Doe) is annoying, but this is the only part that is annoying, we can keep the rest for _at least some_ roleplay value.

It's annoying when you're busy with functional surgeries and someone asks for surgery for this. (Is rare but has happened to me) To quote the wiki "This is a combat operation, cosmetic surgery can wait." - I think this is a good change.

**Comment by ihatethisengine** [comment]
> It's annoying when you're busy with functional surgeries and someone asks for surgery for this. (Is rare but has happened to me) To quote the wiki "This is a combat operation, cosmetic surgery can wait." - I think this is a good change.

You can say this in-character. Or you can perform the surgery if you have time. A lot of stuff doesn't have "functional" purpose, but still has at least some roleplay value. Stale protein bar is the best food source in the game, mechanically, but I I've seen people refusing to eat anything but pizza MRE. People wanting to not look like shit is basically the same. Not wanting to perform a cosmetic surgery just because it's not functional (i.e. doesn't affect any stats) is honestly just powergaming. Well, I guess you can have a character who's that pragmatic. But wanting to remove a mechanic so people won't bother you with cosmetic surgeries is totally just powergaming.

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -113,7 +113,6 @@
 #define CANROOT (1<<6)
 #define GODMODE (1<<12)
 #define FAKEDEATH (1<<13) //Replaces stuff like changeling.changeling_fakedeath
-#define DISFIGURED (1<<14) //I'll probably move this elsewhere if I ever get wround to writing a bitflag mob-damage system
```
Comment (harryob) [inline code/__DEFINES/mobs.dm:116]: make it clear that 1<<14 is unused or move the rest down

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/__DEFINES/mobs.dm (hunk 1 above)
Comment (Doubleumc) [reply code/__DEFINES/mobs.dm:116]: `(1<<14)` is now clearly marked as unused.

**Comment by Doubleumc** [comment]
For what it's worth I looked in to keeping disfigurement and just removing the "as Unknown" part. But... that's _all_ that disfigurement does in this codebase. There's no pizzaface overlay, there's no extra description. Without the "as Unknown" part the only indication you'd have of being disfigured is seeing an extra option in the autodoc.

Additionally there were actually two different systems for disfigurement, one on the mob and one on the head, which worked in slightly different ways and neither of which talked to the other. It was entirely possible to be "disfigured" twice at the same time, each of which needed a different method to heal.

As without the "as Unknown" part it was pointless code and seeing that it was already suffering from lack of maintenance, I am sticking with full removal in this PR.

**Review by harryob — APPROVED** [review]

**Pull request merged**